setup_all.sh

```diff
@@ -4,21 +4,31 @@ source config_vars.sh
 
 set -e
 
+ls config/ | grep default | while read cfgfilename; do
+    new_cfgfilename=${cfgfilename::-8}
+    cp -n config/$cfgfilename config/$new_cfgfilename
+done
+
 # Start MySQL server
 ./run_mysql.sh
 
 # Build and run the API
-docker build -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
+docker build --build-arg API_VERSION="1.0" -t zeeguu-api-core -f docker-files/zeeguu-api-core/Dockerfile .
```

Comment:
@alinbalutoiu : can't we go with API_VERSION=\`cat .git/modules/Zeeguu-API/HEAD]\` here instead?

Reply:
I'm not sure what value should API_VERSION have, but the command you suggested returns: `ref: refs/heads/abtest`.
I've changed API_VERSION to be the hash of the last commit of Zeeguu-API.

Reply:
oki, i like your solution